ui_testing/testcases/base_test.py

```diff
@@ -26,6 +27,7 @@ def setUp(self):
         self.login_page = Login()
         self.order_page = Order()
         self.test_plan = TstPlan()
+        self.testplans_page = TestPlans()
```

Comment:
self.test_plan can access all methods in testplans_page as test_plan inherits from test_plans

Reply:
okey, will remove the import